services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -164,6 +165,10 @@ export const SuccessMessage = ({
   date,
   submittedBy,
 }: SuccessMessageProps) => {
+  // LaunchDarkly feature flags
+  const launchDarklyClient = useLDClient();
+  const pdfExport = launchDarklyClient?.variation("pdfExport", true);
```

Comment:
If we standardize a flag naming convention now it will future tech-debt stories around cleanup easier.

[LD Best Practices](https://docs.launchdarkly.com/guides/flags/technical-debt?site=federal#eliminating-flag-debt)